src/main/java/fi/fmi/avi/converter/tac/lexer/impl/token/SWXPhenonmenonLongitudeLimit.java

```diff
@@ -90,12 +94,12 @@ private String parseLimit(final Double limit) {
             } else {
                 builder.append("E");
             }
-            String[] limtArray = Double.toString(Math.abs(limit)).split("\\.");
-            builder.append(limtArray[0]);
-            builder.append(limtArray[1]);
-            if (limtArray[1].length() < 2) {
-                builder.append("0");
-            }
+            Double absLimit = Math.abs(limit);
+            DecimalFormat formatter = (DecimalFormat) NumberFormat.getNumberInstance();
```

Comment:
During merge I set here fixed `Locale.US` to ensure correct behaviour in different environments.